src/api/flash/DFUProgrammerFlasher.js

```diff
@@ -21,7 +21,47 @@ import path from "path";
 
 import { delay, toStep } from "./utils";
 
-export const DFUProgrammerFlasher = async (board, port, filename, options) => {
+const rebootToNormal = async (port, _) => {
+  logger("flash").debug("rebooting to normal mode");
+  await runDFUProgrammer(["atmega32u4", "start"]);
+};
+
+const runDFUProgrammer = async (args) => {
+  const timeout = 1000 * 60 * 5;
```

Comment:
It's obvious that it's a timeout, can we name it to describe *what* is timing out?